server/game/cards/06-CotE/TheFiresOfJustice.js

```diff
@@ -0,0 +1,46 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, Players, TargetModes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class TheFiresOfJustice extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Remove fate or move fate to a character',
+            when: {
+                afterConflict: (event, context) => event.conflict.winner === context.player && event.conflict.conflictType === 'military'
+            },
+            targets: {
+                character: {
+                    cardType: CardTypes.Character,
+                    player: Players.Opponent,
+                    controller: Players.Opponent,
+                    cardCondition: card => card.isParticipating()
+                },
+                select: {
+                    mode: TargetModes.Select,
+                    dependsOn: 'character',
+                    choices: {
+                        'Remove all fate': AbilityDsl.actions.removeFate(context => ({ target: context.targets.character, amount: context.targets.character.fate })),
+                        'Move fate to character': AbilityDsl.actions.menuPrompt(context => ({
+                            activePromptTitle: 'Select fate amount:',
+                            choices: Array.from(Array(context.player.opponent.fate), (x, i) => (i + 1).toString()),
```

Comment:
You change this to an array of strings - to you need to turn it back to an int when you pass it as a property in `choiceHandler`?